swift/apple/FirezoneKit/Sources/FirezoneKit/Views/SettingsView.swift

```diff
@@ -507,7 +507,8 @@ public struct SettingsView: View {
                 action: {
                   self.isExportingLogs = true
                   Task.detached(priority: .background) { [weak model] in // self can't be weakly captured in views
-                    guard let model else { return }
+                    guard model != nil else { return }
```

Comment:
```suggestion
                    if model == nil {
                    	return
                    }
```

Reply:
So I did a little research on best practices here, and since `guard` enforces early-return, it seems like if that's what we're after then this might be more idiomatic.

Reply:
I.e. the `else` clause of the guard *must* exit the current scope.